README.md

```diff
@@ -16,13 +16,25 @@ It's a Rails engine built on top of [Active Job](https://github.com/rails/active
 
 ## Installation
 
-Add to your Gemfile...
+Add to your Gemfile:
 
 ```ruby
 gem "job_notifier"
 ```
 
-Include `notifier.js` in your `/my-app/app/assets/javascripts/application.js`
+```bash
+bundle install
+```
+
+Run the installer:
+
+```bash
+rails generate job_notifier:install
+```
+
+The installer will copy `/config/initializers/job_notifier.rb`. There, you can change the default configuration.
```

Comment:
tal vez me exprese mal, si entiendo bien lo que hay que hacer yo lo pondria asi (un mini cambio, pero deja mejor las jerarquias)

---

## Installation
  
Add to your Gemfile:
  
```ruby
gem "job_notifier"
```

```bash
bundle install
```

Run the installer:

```bash
rails generate job_notifier:install
```

> The installer will copy `/config/initializers/job_notifier.rb`. There, you can change the default configuration.

Then, include `notifier.js` where you go to use it. For example, in: `/app/assets/javascripts/application.js`

```javascript
//= require job_notifier/notifier
```